localisation/00_zuwc_united_localisation.csv

```diff
@@ -272,13 +272,13 @@ EVTOPTB_WCBDP_80_alliance;One-eyed giants? As if ogres were not enough!;;Ein
+EVTDESC_WCBDP_90_alliance;En route to the Black Temple, several battles have taken place in the forests of Terokkar, one of the few places on Draenor not yet tainted by demonic corruption. This region is home to the draenei, strange creatures fleeing the from orcs for Shattrath City, a walled monastery giving home to some of the enigmatic Naaru.\nIn the woods nearby, a secretive and mysterious bird people, the Arakkoa, have escaped from their homeland that has become occupied by the Shattered Hand. They practice dark magic and make sacrifices to dark gods, akin to Loa-worshipping trolls back on Azeroth. Nevertheless, some of them are in contact with other races.\nIf possible, perhaps one of them will agree to help us in our upcoming battles?;;Auf dem Weg zum Schwarzen Tempel haben wir viele Schlachten in Terokkar geschlagen, einem der wenigen Orte auf Draenor, die bisher noch nicht von d�monischer Magie verseucht wurden. Dieser Ort ist die Heimat der Draenei, bizarrer Kreaturen die vor den Orks in die Stadt Shattrath fl�chten, ein ummauertes Kloster, in dem die r�tselhaft Naaru leben.\nIn den nahen W�ldern leben die geheimnisvollen und verschlossenen Vogelleute, die Arakkoa, seit sie aus aus ihrer Heimat geflohen sind, die nun vom Clan der zerschmetterten Hand besetzt gehalten wird. Sie praktizieren finstere Magie und bringen ihren dunklen G�ttern Opfer dar, ganz so wie es die Loa-anbetenden Trolle tun. Dennoch meiden einige von ihnen den Kontakt mit anderen Rassen nicht.\nVielleicht wird einer von ihnen sogar zustimmen, uns in der bevorstehenden Schlacht zu unterst�tzen?;;;;;;;;;;;x
```

Comment:
German: "Auf dem Weg zum Schwarzen Tempel fanden mehrere Schlachten in den Wäldern von Terokkar statt, einem der wenigen Orte auf Draenor, der noch nicht von dämonischer Magie verseucht wurden. In dieser Region leben die Draenei, seltsame Kreaturen, die vor den Orks in die Stadt Shattrath geflohen sind, ein ummauertes Kloster, in dem die rätselhaften Naaru leben. \nIn den nahen Wäldern leben die geheimnisvollen und verschlossenen Vogelleute, die Arakkoa, seit sie aus aus ihrer Heimat geflohen sind, die nun vom Clan der zerschmetterten Hand besetzt gehalten wird. Sie praktizieren dunkle Magie und opfern dunklen Götter, ähnlich wie die Loa-verehrenden Trolle auf Azeroth. Dennoch treten einige von ihnen in Kontakt mit anderen Rassen. \nVielleicht wird einer von ihnen sogar zustimmen, uns in der bevorstehenden Schlacht zu unterstützen?"